Documentation/proposal-runconfig_inside_osv.md

```diff
@@ -0,0 +1,40 @@
+# Proposal for meta/run.yaml content being present in the composed image
+
+Currently, Capstan runtime support makes use of `$CWD/meta/run.yaml` for composing and running an OSv image. Although this is fine for the composing, it gets tricky for running because `capstan run <IMAGE>` generally does not require user to be inside package root directory.
+
+I propose the following solution:
+
+### Composing image
+File meta/run.yaml is present at package root directory (same as until now)
+
+When user runs `capstan package compose` the content of meta/run.yaml file is processed and new files are generated inside the OSv image folder `/osv-runtime`:
+
+Capstan should create a separate file for each named configuration e.g.
+```
+/osv-runtime
+   |--burek
+   |--pizza   <--------\
+   |                    |
+   |--default-runtime  /
+```
+Each runtime file should contain only one line representing the run command that is to be run inside OSv:
+```
+# content of file /osv-runtime/burek:
+node burek.js
```

Comment:
The example says node burek.js: does this require Agent to know about how to run the Node.JS server or is this something Capstan will work out? For example, the command will actually be /node burek.js (since we have a symlink /node to /bin/libnode-4.4.5.so in the Node.JS package?